src/screenshot.cpp

```diff
@@ -48,7 +48,7 @@ uint _heightmap_highest_peak;         ///< When saving a heightmap, this contain
  */
 static ScreenshotProvider *GetScreenshotProvider()
 {
-	auto providers = ProviderManager<ScreenshotProvider>::GetProviders();
+	auto &providers = ProviderManager<ScreenshotProvider>::GetProviders();
```

Comment:
const? Or no difference?

Reply:
Making this `const` requires changing all the `ScreenshotProviders` to make `MakeImage` a `const` function. That's probably worth it, but not necessarily for this PR.